### DISCUSSION TITLE
Counter Not Returned

### DISCUSSION THREAD

**Discussion by nishils** [MasterKale/SimpleWebAuthn#376] [Troubleshooting]
Hi there,

I am using the latest versions 7.2.0 on both client and server. Upon digging into the client's response to verify registration, I don't see a counter value being returned in the request to my server.

Do we assume that the absence of a counter indicates the value should be 0?

**Comment by MasterKale** [comment]
@nishils You found a resolution for this? Was it the `counter` value you were looking for in the output from `verifyRegistrationResponse()` and `verifyAuthenticationResponse()`?
